cmd/kafka-consumer/main.go

```diff
@@ -574,9 +579,20 @@ func (c *Consumer) forEachSink(fn func(sink *partitionSink) error) error {
 	return nil
 }
 
-// Run runs the Consumer
+func (c *Consumer) getMinPartitionResolvedTs() (result uint64, err error) {
+	result = uint64(math.MaxUint64)
+	err = c.forEachSink(func(sink *partitionSink) error {
+		a := atomic.LoadUint64(&sink.resolvedTs)
```

Comment:
change `a`  to a readable name？

Reply:
it's only a local variable, does not affect out scope, right ?